packages/ui/tailwind-design-system.ts

```diff
@@ -620,7 +622,6 @@ export default {
     // Hover classes
     { pattern: /^hover:bg-graph-/ },
     'stroke-borders-2',
-    { pattern: /rounded-./ },
```

Comment:
is this not needed now? What if we use rounded classes which we didn't use in UI package but in other applications that is using UI package? In this case, will the unused classes defined in config also not gets added as a part of the style bundle, or it will be added?